tests/unit-core/test-typedarray.c

```diff
@@ -303,6 +303,102 @@ test_typedarray_complex_creation (test_entry_t test_entries[], /**< test cases *
   }
 } /* test_typedarray_complex_creation */
 
+/**
+ * Test get/set/delete property by index.
+ */
+static void test_property_by_index (test_entry_t test_entries[])
+{
+  for (uint32_t i = 0; test_entries[i].constructor_name != NULL; i++)
+  {
+    jerry_value_t test_numbers[5] =
+    {
+      jerry_create_number (120),
+      jerry_create_number (55),
+      jerry_create_number (11),
+      jerry_create_number (0),
+      jerry_create_number (5)
+    };
+    uint32_t test_numbers_length = sizeof (test_numbers) / sizeof (jerry_value_t);
+    jerry_value_t typedarray = jerry_create_typedarray (test_entries[i].typedarray_type, test_numbers_length);
+    jerry_typedarray_type_t type = jerry_get_typedarray_type (typedarray);
+
+    switch (type)
+    {
+      case JERRY_TYPEDARRAY_INT8:
+      case JERRY_TYPEDARRAY_INT16:
+      case JERRY_TYPEDARRAY_INT32:
+      for (uint32_t j = 0; j < test_numbers_length; j+=2)
+      {
+        test_numbers[j] = jerry_create_number (0 - jerry_get_number_value (test_numbers[j]));
```

Comment:
What happens with the original number? I know you don't need to free them, but still you should according the documentation.

Btw, you could have a simple const number list, not an ecma_value_t list. This way you can avoid the release problem.